scripts/superdesk-desks/views/desk-config-modal.html

```diff
@@ -41,7 +41,7 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
                                 <label translate>Desk Type</label>
                                 <select id="deskType" ng-model="desk.edit.desk_type" required>
                                     <option value=""></option>
-                                    <option id="{{type.value}}" ng-repeat="type in deskTypes" value="{{type.value}}" ng-selected="desk.edit.desk_type === type.value"
+                                    <option id="{{type.qcode}}" ng-repeat="type in deskTypes" value="{{type.qcode}}" ng-selected="desk.edit.desk_type === type.qcode"
```

Comment:
maybe you can make it compatible with both versions? like `{{ type.qcode || type.value }}`